src/pages/dashboard/_CDCComparisonContainer.js

```diff
@@ -1,64 +1,101 @@
+  if (a.date > b.date) return 1
+  if (a.date < b.date) return -1
+  return 0
+}
 
 export default function CDCComparisonContainer() {
   const query = useStaticQuery(graphql`
     {
       allCovidUsDaily {
         nodes {
           date
-          positive
-          negative
+          totalTestResultsIncrease
+        }
+      }
+      allCdcDaily {
+        nodes {
+          dateCollected
+          lag
+          cdcLabs
+          dailyTotal
         }
       }
     }
   `)
-  const data = useMemo(() => {
-    const nodes = query.allCovidUsDaily.nodes
-      .map(node => [
-        {
-          date: parseDate(node.date),
-          value: calculateTotal(node),
-        },
-      ])
-      .flat()
-
-    return nodes.sort((a, b) => {
-      if (a.date > b.date) return 1
-      if (a.date < b.date) return -1
-      return 0
+  const cdcData = query.allCdcDaily.nodes
+    .map(node => {
+      const date = parseCdcDate(`${node.dateCollected}/2020`)
+      return {
+        date,
+        value: +node.dailyTotal,
+      }
     })
-  }, [query.allCovidUsDaily.nodes.length])
+    .sort(sortChronologically)
+  const data = query.allCovidUsDaily.nodes
+    .map(node => [
+      {
+        date: parseDate(node.date),
+        value: node.totalTestResultsIncrease,
+      },
+    ])
+    .flat()
+    .sort(sortChronologically)
+  const cdcCumulativeTotal = sum(cdcData, d => d.value)
   const cumulativeTotal = sum(data, d => d.value)
+  const dailyMax = max(data, d => d.value)
   return (
     <section style={{ display: 'flex' }}>
       <div style={{ flexGrow: 1, width: '50%' }}>
         <h4>Differences between this data and the CDC data</h4>
         <p>
-          As of today, the C.D.C. has tested 133,442 specimens, and we have
-          tracked at least {formatNumber(cumulativeTotal)} tests administered
-          across the country. It is important to note that testing numbers are
-          likely an undercount because of the lack of universal testing and
-          state reporting, and that multiple specimen tests can be conducted
-          from a single person’s sample.
+          As of today, the C.D.C. has tested {formatNumber(cdcCumulativeTotal)}
+          specimens, and we have tracked at least{' '}
+          {formatNumber(cumulativeTotal)} tests administered across the country.
+          It is important to note that testing numbers are likely an undercount
+          because of the lack of universal testing and state reporting, and that
+          multiple specimen tests can be conducted from a single person’s
+          sample.
         </p>
       </div>
-      <div style={{ flexGrow: 1, width: '50%' }}>
-        <BarChart
-          data={data}
-          fill="#585BC1"
-          height={400}
-          marginBottom={40}
-          marginLeft={80}
-          marginRight={10}
-          marginTop={10}
-          xTicks={2}
-          width={200}
-        />
+      <div style={{ display: 'flex', flexGrow: 1, width: '50%' }}>
```

Comment:
All this inline style business is going to have to change, but just wanted to get this up quickly